pabi_chartfield/models/chartfield.py

```diff
@@ -636,10 +636,12 @@ def _get_default_fund(self):
         if self.invest_construction_phase_id:
             funds = self.invest_construction_phase_id.fund_ids
         # Get default fund
-        if len(funds) == 1:
-            fund_id = funds[0].id
-        else:
-            fund_id = False
+        # if len(funds) == 1:
+        #     fund_id = funds[0].id
+        # else:
+        #     fund_id = False
+        # Change from Issue 4373
+        fund_id = funds[0].id

```

Comment:
What if funds is nothing. Index 0 will give error?

Reply:
I read a desciption 
"# If that dimension have 1 funds, use that fund.
 # If that dimension have no funds, use NSTDA
 # Else return false"

I thought if funds is noting will return NSTDA  someway
but I fixed It